osu.Game/OsuGame.cs

```diff
@@ -549,6 +550,17 @@ public void PresentScore(IScoreInfo score, ScorePresentType presentType = ScoreP
             if (score is ScoreInfo scoreInfo)
                 databasedScoreInfo ??= ScoreManager.Query(s => s.Hash == scoreInfo.Hash);
 
+            if (!(score.User.Username is null))
```

Comment:
This seems like a very weird place to do this lookup. Is there a reason it's not done as part of the import process instead?

Reply:
Already mentioned that in thread as well, `ScoreImporter.PostImport` should be a good place for such logic.

Reply:
> This seems like a very weird place to do this lookup. Is there a reason it's not done as part of the import process instead?

I couldn't find an effective way to import the API to `PostImport`, which is why I'm sending the api request here.